app/serializers/doorkeeper/token_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  class TokenSerializer
+    include JSONAPI::Serializer
+
+    attributes :token, :token_type, :expires_in, :refresh_token
+
+    attribute :created_at do |token|
+      token.created_at.to_time.to_i
```

Comment:
How about directly calling `.to_i`, why do we need to convert to time

```
   token.created_at.to_i
```

Reply:
Copy/paste from doorkeeper tutorial >< 
I just removed it :)